### ISSUE TITLE
Failures during OCS-CI teardown

### ISSUE THREAD

**Issue by shylesh** [red-hat-storage/ocs-ci#3683]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-destroy-ocs-cluster/12801 .

======================================
16:56:39 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get Service  -n openshift-storage --selector=app=rook-ceph-rgw -o yaml
16:56:39 - MainThread - tests.conftest - INFO - RGW service is not available
16:56:39 - MainThread - /home/jenkins/workspace/qe-destroy-ocs-cluster/ocs-ci/ocs_ci/ocs/resources/cloud_manager.py - INFO - Trying to load credentials from ocs-ci-data. This flow is only relevant when running under OCS-QE environments.
16:56:39 - MainThread - /home/jenkins/workspace/qe-destroy-ocs-cluster/ocs-ci/ocs_ci/utility/aws.py - INFO - Fetching authentication credentials from ocs-ci-data
16:56:40 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Secret with name secret-cldmgr-aws-1fe958c8dd7b4bfcb26090
16:56:40 - MainThread - ocs_ci.utility.templating - INFO - apiVersion: v1
data:
  AWS_ACCESS_KEY_ID: '*****'
  AWS_SECRET_ACCESS_KEY: '*****'
kind: Secret
metadata:
  name: secret-cldmgr-aws-1fe958c8dd7b4bfcb26090
  namespace: openshift-storage
type: Opaque

16:56:40 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage create -f /tmp/Secretcud9b74s -o yaml
16:56:40 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: Error from server (NotFound): error when creating "/tmp/Secretcud9b74s": namespaces "openshift-storage" not found

ERROR                                                                    [100%]

======================================

Strange that namespace is missing, it worked in the first command and suddenly not available.

______________________ ERROR at setup of test_deployment _______________________
```python
request = <SubRequest 'cld_mgr' for <Function test_deployment>>
rgw_endpoint = None

    @pytest.fixture(scope="session")
    def cld_mgr(request, rgw_endpoint):
        """
        Returns a cloud manager instance that'll be used throughout the session
    
        Returns:
            CloudManager: A CloudManager resource
    
        """
>       cld_mgr = CloudManager()

tests/conftest.py:1639: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/resources/cloud_manager.py:83: in __init__
    cloud_map[cloud_name](auth_dict=cred_dict[cloud_name]),
ocs_ci/ocs/resources/cloud_manager.py:199: in __init__
    self.secret = self.create_s3_secret(secret_prefix)
ocs_ci/ocs/resources/cloud_manager.py:351: in create_s3_secret
    return create_resource(**bs_secret_data)
ocs_ci/helpers/helpers.py:82: in create_resource
    created_resource = ocs_obj.create(do_reload=do_reload)
ocs_ci/ocs/resources/ocs.py:103: in create
    status = self.ocp.create(yaml_file=self.temp_yaml)
ocs_ci/ocs/ocp.py:301: in create
    output = self.exec_oc_cmd(command)
ocs_ci/ocs/ocp.py:144: in exec_oc_cmd
    **kwargs,
ocs_ci/utility/utils.py:451: in run_cmd
    completed_process = exec_cmd(cmd, secrets, timeout, ignore_error, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cmd = ['oc', '-n', 'openshift-storage', 'create', '-f', '/tmp/Secretcud9b74s', ...]
secrets = None, timeout = 600, ignore_error = False, kwargs = {}
masked_cmd = 'oc -n openshift-storage create -f /tmp/Secretcud9b74s -o yaml'
completed_process = CompletedProcess(args=['oc', '-n', 'openshift-storage', 'create', '-f', '/tmp/Secretcud9b74s', '-o', 'yaml'], returnco...b'Error from server (NotFound): error when creating "/tmp/Secretcud9b74s": namespaces "openshift-storage" not found\n')
masked_stdout = ''
masked_stderr = 'Error from server (NotFound): error when creating "/tmp/Secretcud9b74s": namespaces "openshift-storage" not found\n'

```

This could be a relative of **https://github.com/red-hat-storage/ocs-ci/issues/3675**

**Comment by vavuthu** [comment]
This is addressed in PR https://github.com/red-hat-storage/ocs-ci/pull/3681/files

**Comment by petr-balogh** [comment]
Closing as fix #3681 is merged
